k2/csrc/fsa_utils.h

```diff
@@ -527,6 +527,21 @@ FsaVec FsaVecFromArcIndexes(FsaVec &fsas, Ragged<int32_t> &best_arc_indexes);
 Ragged<int32_t> ComposeArcMaps(Ragged<int32_t> &step1_arc_map,
                                Ragged<int32_t> &step2_arc_map);
 
+/*
+  Compose arc maps from two successive FSA operations which give arc maps as
+  a 1-D array.
```

Comment:
I notice the Index() function in array_ops.h is the same as this except it sets the value to 0 instead of -1.
How about adding an extra arg to that function that is the value to be used if the index is -1, defaulting to 0?
I prefer to use general-purpose operations.

Reply:
Yes, I agree.

Reply:
Fixed.